src/main/java/io/github/queerbric/inspecio/tooltip/EntityTooltipComponent.java

```diff
@@ -103,6 +105,8 @@ protected void renderEntity(MatrixStack matrices, int x, int y, Entity entity, i
 		this.setupAngles(entity, this.client.player.age, ageOffset, spin, defaultYaw);
 		var entityRenderDispatcher = this.client.getEntityRenderDispatcher();
 		quaternion2.conjugate();
+		((CameraAccessor) entityRenderDispatcher.camera).setYaw(0f);
```

Comment:
You confirm that it doesn't mess the player's in-world view?

Reply:
no, it does not and only affects the vision of the fire